net.ptilouk.superfluous.yml

```diff
@@ -0,0 +1,117 @@
+        sha256: 2a026753af9b03fca043824bca8341f74921a836d28729e0c31aa262202a83c6
+        
+  - name: SDL_Mixer_X
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/WohlSoft/SDL-Mixer-X
+        commit: 7181c771aa29deaea8bfbe6098613c3a929e9189
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DUSE_DRFLAC:BOOL=OFF
+      - -DUSE_GME:BOOL=OFF
+      - -DUSE_MIDI:BOOL=OFF
+      - -DUSE_MIDI_ADLMIDI:BOOL=OFF
+      - -DUSE_MIDI_EDMIDI:BOOL=OFF
+      - -DUSE_MIDI_FLUIDLITE:BOOL=OFF
+      - -DUSE_MIDI_OPNMIDI:BOOL=OFF
+      - -DUSE_MIDI_TIMIDITY:BOOL=OFF
+      - -DUSE_MODPLUG:BOOL=OFF
+      - -DUSE_MP3_DRMP3:BOOL=OFF
+      - -DUSE_OGG_VORBIS:BOOL=ON
+      - -DUSE_OGG_VORBIS_STB:BOOL=ON
+      - -DUSE_OPUS:BOOL=OFF
+      - -DUSE_PXTONE:BOOL=OFF
+      - -DUSE_SYSTEM_SDL2:BOOL=ON
+      - -DUSE_WAV:BOOL=OFF
+      - -DUSE_WAVPACK:BOOL=OFF
+
+  - name: SDL_ttf
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_ttf/archive/refs/tags/release-2.20.2.tar.gz
+        sha256: 0fe9d587cdc4e6754b647536d0803bea8ca6ac77146c4209e0bed22391cf8241
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+
+  - name: libyaml
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/yaml/libyaml/archive/refs/tags/0.2.5.tar.gz
+        sha256: fa240dbf262be053f3898006d502d514936c818e422afdcf33921c63bed9bf2e
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+
+  - name: lz4
+    buildsystem: simple
+    build-commands:
+      - make
+      - make install PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/lz4/lz4/archive/refs/tags/v1.9.3.tar.gz
+        sha256: 030644df4611007ff7dc962d981f390361e6c97a34e5cbc393ddfbe019ffe2c1
+
+  - name: superfluous-returnz-data
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/superfluous-returnz
+      - cp -r * /app/share/superfluous-returnz
+    sources:
+      - type: git
+        url: https://github.com/sgiraudot/superfluous-returnz-data
+        commit: 9ee8e318287141079c1ad95d498d6f52738916f3
+
+  - name: superfluous-returnz
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSOSAGE_BUILD_TYPE=Release
+      - -DSOSAGE_CFG_DISPLAY_DEBUG_INFO:BOOL=False
+      - -DSOSAGE_DATA_FOLDER=/app/share/superfluous-returnz
+      - -DSOSAGE_CFG_USE_SDL_MIXER_EXT:BOOL=True
+    sources:
+      - type: git
+        url: https://github.com/sgiraudot/sosage
+        commit: 266df855e3f22db1cb8f64514b502d9ac6dfaa0a
+
+  - name: move-icon
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - mv /app/share/icons/superfluous-returnz.svg /app/share/icons/hicolor/scalable/apps/superfluous-returnz.svg
```

Comment:
add it to `post-install` of above or whatever install the superfluous-returnz.svg file